src/Eagle.ts

```diff
@@ -408,13 +409,18 @@ export class Eagle {
     }
 
     setSelection = (rightWindowMode : Eagle.RightWindowMode, selection : Node | Edge) : void => {
-        //console.log("eagle.setSelection()", Utils.translateRightWindowModeToString(rightWindowMode), selection);
+        //set amount of collapsed node menus to default (0)
 
+        $(".nodeInspectorCollapseAll").collapse("hide");
+        $(".nodeMenuIndicator").addClass("closedIcon");
```

Comment:
These calls to addClass and removeClass were what initially made me think of trying a knockout-like approach. Knockout has a data-bind specifically for this purpose.

data-bind="css: {className: conditional-to-evaluate}".

If the conditional evaluates as true, then knockout adds the class (className) to the element, otherwise it removes the class.